buttons.js

```diff
@@ -27,6 +27,7 @@ const buttons = {
         new ButtonBuilder()
             .setCustomId('self_hosted')
             .setLabel('Self Hosted server')
+            .setEmoji("<:log:1096054676903956560>")
```

Comment:
instead of adding it like that would you mind sending us the emoji so we can upload it and give u the id?

Reply:
it might, might not. Better to be safe than sorry. I can send u the id after anyway

Reply:
o wait its a literal log lmao from the qc server

Reply:
could you maybe get a custom emoji for a self hosted server guide?

Reply:
https://github.com/Cartrigger/QCXRSoonBot/pull/51/commits/1f9d5b01a36fe9fc6693ee4a6bd2f9c66be75fa3 Fixed it :)